lua/cps.lua

```diff
@@ -0,0 +1,28 @@
+return function (net_conf, numa_table)
+
+	-- Init the CPS configuration structure.
+	local cps_conf = gatekeeper.c.get_cps_conf()
+	if cps_conf == nil then
+		error("Failed to allocate cps_conf")
+	end
+
+	-- Change these parameters to configure the CPS block.
+	local kni_kmod_path =
+		"/home/cody/gatekeeper/dependencies/dpdk/build/kmod/rte_kni.ko"
```

Comment:
Move this single parameter to the beginning of the function. There's nothing else that casual users would need to edit here. Adding a comment to say that as we've done in the files of other blocks.